runtime/onert/core/include/exec/Executors.h

```diff
@@ -20,6 +20,21 @@
 #include "IExecutor.h"
 #include "ir/NNPkg.h"
 
+namespace std
+{
+
+template <> struct hash<std::pair<::onert::ir::ModelIndex, ::onert::ir::SubgraphIndex>>
+{
+  size_t
+  operator()(const std::pair<::onert::ir::ModelIndex, ::onert::ir::SubgraphIndex> &pair) const
+    noexcept
+  {
+    return hash<uint64_t>()((uint64_t(pair.first.value()) << 32) | uint64_t(pair.second.value()));
```

Comment:
(optional) I didn't consider arm32 when I suggested 64bit hash. We may use 32bit version. ( But I don't think it matters. )

```suggestion
    return hash<uint32_t>()((pair.first.value() << 16) | pair.second.value());
```

Reply:
I'll make PR to change `ModelIndex` and `SubgraphIndex` value type to `uint16_t` later.

Reply:
How about use XOR operator?
```suggestion
    auto hash1 = hash<uint32_t>{}(pair.first.value());
    auto hash2 = hash<uint32_t>{}(pair.second.value());
 
    if (hash1 != hash2) {
      return hash1 ^ hash2;             
    }
         
    // If hash1 == hash2, their XOR is zero.
    return hash1;
```

Reply:
Let's update this when we change value type.

Reply:
Hmm. Is it safe to use hash value shift-xor?

Reply:
If both(hash1 and shifted hash2) are not the same, it is safe.

Reply:
And If hash2 >= 2^16, it is not safe?